src/lib/input_schema.js

```diff
@@ -65,6 +71,42 @@ const readInputSchema = async (forcePath) => {
     };
 };
 
+/**
+ * Goes to the Actor directory and creates INPUT.json file from the input schema prefills.
+ * @param {string} actorFolderDir
+ */
+const createPrefilledInputFileFromInputSchema = async (actorFolderDir) => {
+    const currentDir = process.cwd();
+    let inputFile = {};
+    try {
+        process.chdir(actorFolderDir);
+        const { inputSchema } = await readInputSchema();
+
+        if (inputSchema) {
+            /**
+             * TODO: The logic is copied from @apify-packages/actor -> getPrefillFromInputSchema
+             * It is not possible to install the package here because it is private
+             * We should move it to @apify/input_schema and use it from there.
+             */
+            const validator = new Ajv({ strict: false });
+            validateInputSchema(validator, inputSchema);
+
+            inputFile = _.mapObject(inputSchema.properties, (fieldSchema) => ((fieldSchema.type === 'boolean' || fieldSchema.editor === 'hidden')
+                ? fieldSchema.default
+                : fieldSchema.prefill
+            ));
+        }
+    } catch (err) {
+        warning(`Could not create INPUT.json file. Cause: ${err.message}`);
```

Comment:
The warning isn't really correct now, it should say something like "Could not create default input based on input schema, creating empty input instead" or similar